pkg/chart/ChartService.go

```diff
@@ -487,7 +487,7 @@ func (impl ChartServiceImpl) CreateChartFromEnvOverride(templateRequest Template
 		return nil, err
 	}
 	var appLevelMetrics *repository3.AppLevelMetrics
-	if appMetrics && !(chartMajorVersion >= 3 && chartMinorVersion >= 1) {
+	if appMetrics && !(chartMajorVersion >= 3 && chartMinorVersion >= 7) {
 		impl.logger.Error("cannot enable app metrics for older chart versions < 3.1.0")
```

Comment:
versions < 3.7.0

Reply:
we extract version number from string because is there no support in our code currently to compare version strings.